RedacteurPortaal.Api/Controllers/NewsItemController.cs

```diff
@@ -68,6 +68,62 @@ public async Task<ActionResult<List<NewsItemDto>>> GetNewsItems()
         return this.Ok(response);
     }
 
+    [HttpGet]
+    [Route("sortByDate/{startDate}/{endDate}", Name = "SortNewsItemByDateTime")]
+    public async Task<ActionResult<List<NewsItemDto>>> SortNewsItemByDateTime(DateTime startDate, DateTime endDate)
+    {
+        var grain = await this.grainService.GetGrains();
+        
+        var list = (await grain.SelectAsync(async x => await
+        x.Get())).AsQueryable().ProjectToType<NewsItemDto>(null).ToList();
+
+        var response = list.Where(x => x.ProductionDate >= startDate && x.ProductionDate <= endDate).ToList();
```

Comment:
Je gaat nu van een queryable naar een lijst van dto's waarna die weer intern naar een queryable gaat voor die where en m dan weer mapped naar een lijst.

Naar queryable en type mag als laatste dat we dat maar 1x doen,